rmgpy/molecule/moleculeTest.py

```diff
@@ -2144,6 +2144,50 @@ def testFingerprintProperty(self):
         self.molecule[0].fingerprint = 'nitronate'
         self.assertEqual(self.molecule[0].fingerprint, 'nitronate')
 
+    def testSaturatedUnfilleValence(self):
```

Comment:
I think you added the 'd' to the wrong place. It should be `testSaturateUnfilledValence`.

Reply:
thanks :)
fixed